pkg/controller/checker/checker.go

```diff
@@ -8,32 +8,49 @@ import (
 	corev1 "k8s.io/api/core/v1"
```

Comment:
Not a fan of how I have organised the packages here. I think it would be better to change it to;

```
/pkg/controller
/pkg/controller/nodes
/pkg/controller/pods
/pkg/controller/scheduler
/pkg/controller/options
/pkg/checker
/pkg/checker/search
/pkg/checker/architecture
/pkg/checker/version
```